daffodil-core/src/main/scala/edu/illinois/ncsa/daffodil/compiler/Compiler.scala

```diff
@@ -67,6 +67,7 @@ import edu.illinois.ncsa.daffodil.processors.parsers.NotParsableParser
 import edu.illinois.ncsa.daffodil.processors.unparsers.NotUnparsableUnparser
```

Comment:
This is to figure out the difference between a review, and just comments on a patch set.